packages/shared/lib/services/user.service.ts

```diff
@@ -51,8 +51,12 @@ class UserService {
         return result[0] || null;
     }
 
-    async getUsersByAccountId(accountId: number): Promise<DBUser[]> {
-        const result = await db.knex.select('*').from<DBUser>(`_nango_users`).where({ account_id: accountId, suspended: false });
+    async getUsersByAccountId(accountId: number, requireEmail: boolean = false): Promise<DBUser[]> {
+        let query = db.knex.select('*').from<DBUser>(`_nango_users`).where({ account_id: accountId, suspended: false });
+        if (requireEmail) {
+            query = query.orderBy('created_at', 'asc').limit(1);
+        }
```

Comment:
the `if` does not match the query condition. I'm not sure what it's really getting here.